proposal.md

```diff
@@ -22,9 +22,8 @@ In a few sentences, provide some motivation for why you are creating a dashboard
 
 Example:
 
-> Understanding how the variety of barley and the site on which they grow impact the annual yield is very important. It can be very useful for farmers who want to have the highest yields as possible, but is can also help agronomists to do their job. 
+> Barley is part of the major cereal grains used worldwide. Understanding how the variety of barley and the site on which they grow impact the annual yield is very important. It can help farmers to have the highest yield as possible and increase their productivity, but is can also help agronomists to have a better understanding of how the environment impacts each variety of barley. To help those farmers increasing their productivity, we decided to create an app that could allow them to explore a dataset containing information on annual yields for different sites and barley varieties. This app will allow the users to see the annual yield for selected varieties and particular sites, for the years 1931, 1932, or both. It should help them understand what variety or what site is the more suitable to their situation.
```

Comment:
```suggestion
> Barley is part of the major cereal grains used worldwide. Understanding how the variety of barley and the site on which it grows impacts the annual yield is very important. It can help farmers to have the highest yield as possible and increase their productivity, but it can also help agronomists to have a better understanding of how the environment impacts each variety of barley. To help those farmers increase their productivity, we decided to create an app that could allow them to explore a dataset containing information on annual yields for different sites and barley varieties. This app will allow the users to see the annual yield for selected varieties and particular sites, for the years 1931, 1932, or both. It should help them understand what variety or what site is the most suitable to their situation.
```

Reply:
Just some suggested changes regarding grammar/spelling. Please feel free to revert if you disagree!